src/desktop/apps/search/client/index.coffee

```diff
@@ -29,8 +29,6 @@ module.exports.SearchResultsView = class SearchResultsView extends Backbone.View
         else if result.display_model is 'Artwork'
           @refreshRenderArtworks result
 
-    splitTest('new_search_page').view()
```

Comment:
The import on [line 10](https://github.com/artsy/force/pull/3995/files#diff-1258efd7fc4474941ba763c5690dfeadR10) could also be removed, if this is the only reference. (Which I think it is...)